.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+Thank you for taking the time to contribute to OpenRewrite!
+Feel free to delete any sections that don't apply to your pull request.
+-->
+
+## What's changed?
+<!-- A brief description of the changes in this pull request -->
+
+## What's your motivation?
+<!-- This can link to close a separate issue, or be described on the pull request itself -->
+Fixes #1
+
+## Anything in particular you'd like reviewers to focus on?
+<!-- You can also start a discussion on particular aspects of your implementation on the files tab yourself. -->
+
+## Anyone you would like to review specifically?
+<!-- @mention them here -->
+
+## Have you considered any alternatives or workarounds?
+<!-- Any other ways to solve the problem, or ways to work around the problem. -->
+
+## Any additional context
+<!-- Any thoughts you would like to share in addition to the above. -->
+
+### Checklist
+
+- [ ] I've added unit tests to cover both positive and negative cases
+- [ ] I've added the license header to any new files through `./gradlew licenseFormat`
+- [ ] I've used the IntelliJ auto-formatter on affected files
```

Comment:
When I run auto-format in my IntelliJ, I see quite a few changes in rewrite. Is there some way of standardizing this or ensuring people don't have competing formats?

Reply:
Yah I think we don't expect people to format all files; only the changed file. That standardizes things a bit at least, until we integrate automated code format suggestions through the CLI.